tests/edit_parameters.cc

```diff
@@ -39,7 +40,9 @@ namespace aspect
     if ( simulator_access.get_timestep_number() >= switch_step && !switched )
       {
         simulator_access.get_pcout()<<"Switching boundary condition!"<<std::endl;
-        parameters.fixed_temperature_boundary_indicators.erase(bottom);
+        std::set<types::boundary_id> &temperature_boundary_ids =
+          const_cast<std::set<types::boundary_id> &> (simulator_access.get_boundary_temperature_manager().get_fixed_temperature_boundary_indicators());
+        temperature_boundary_ids.erase(bottom);
```

Comment:
oh, if this was anywhere other than in a test, I'd be quite unhappy about such a trick :-)